terraform/task-definitions/publisher.json

```diff
@@ -0,0 +1,114 @@
+      { "name": "BASIC_AUTH_USERNAME", "value": "gds" },
+      { "name": "EMAIL_GROUP_BUSINESS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_CITIZEN", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_DEV", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "EMAIL_GROUP_FORCE_PUBLISH_ALERTS", "value": "test-address@digital.cabinet-office.gov.uk" },
+      { "name": "FACT_CHECK_SUBJECT_PREFIX", "value": "dev"},
+      { "name": "FACT_CHECK_USERNAME", "value": "govuk-fact-check-test@digital.cabinet-office.gov.uk"},
+      { "name": "GOVUK_APP_DOMAIN", "value": "test.govuk-internal.digital" },
+      { "name": "GOVUK_APP_NAME", "value": "publisher"},
+      { "name": "GOVUK_APP_TYPE", "value": "rack"},
+      { "name": "GOVUK_STATSD_PREFIX", "value": "fargate" },
+      { "name": "GOVUK_ASSET_ROOT", "value": "https://assets.digital.cabinet-office.gov.uk" },
+      { "name": "GOVUK_GROUP", "value": "deploy"},
+      { "name": "GOVUK_USER", "value": "deploy"},
+      { "name": "GOVUK_WEBSITE_ROOT", "value": "https://www.gov.uk" },
+      { "name": "PLEK_SERVICE_CONTENT_STORE_URI", "value": "https://www.gov.uk/api" },
+      { "name": "PLEK_SERVICE_STATIC_URI", "value": "https://assets.publishing.service.gov.uk" },
+      { "name": "RAILS_ENV", "value": "production" },
+      { "name": "RAILS_SERVE_STATIC_FILES", "value": "true" },
+      { "name": "REDIS_HOST", "value": "pink-backend-redis.0f3erf.ng.0001.euw1.cache.amazonaws.com"},
+      { "name": "REDIS_PORT", "value": "6379"},
+      { "name": "REDIS_URL", "value": "redis://pink-backend-redis.0f3erf.ng.0001.euw1.cache.amazonaws.com:6379"},
+      { "name": "STATSD_PROTOCOL", "value": "tcp" },
+      { "name": "STATSD_HOST", "value": "statsd.test.govuk-internal.digital" },
+      { "name": "WEBSITE_ROOT", "value": "test.publishing.service.gov.uk" }
+    ],
+    "logConfiguration": {
+        "logDriver": "awslogs",
+        "options": {
+            "awslogs-create-group": "true",
+            "awslogs-group": "awslogs-fargate",
+            "awslogs-region": "eu-west-1",
+            "awslogs-stream-prefix": "awslogs-publisher"
+        }
+    },
+    "mountPoints": [],
+    "portMappings": [
+      {
+        "containerPort": 3000,
+        "hostPort": 3000,
+        "protocol": "tcp"
+      }
+    ],
+    "secrets": [
+      {
+        "name": "FACT_CHECK_PASSWORD",
+        "valueFrom": "arn:aws:secretsmanager:eu-west-1:430354129336:secret:publisher_app-FACT_CHECK_PASSWORD-A2UM6N"
+      },
+      {
+        "name": "FACT_CHECK_REPLY_TO_ADDRESS",
+        "valueFrom": "arn:aws:secretsmanager:eu-west-1:430354129336:secret:publisher_app-FACT_CHECK_REPLY_TO_ADDRESS-zZ9I9x"
+      },
+      {
+        "name": "FACT_CHECK_REPLY_TO_ID",
+        "valueFrom": "arn:aws:secretsmanager:eu-west-1:430354129336:secret:publisher_app-FACT_CHECK_REPLY_TO_ID-c725KL"
+      },
```

Comment:
I don't think these two need to be secrets (FACT_CHECK_REPLY_TO_ADDRESS and FACT_CHECK_REPLY_TO_ID). They're currently [published in alphagov/govuk-puppet](https://github.com/alphagov/govuk-puppet/blob/502262ba5e81287d361a84281c3386801c45d6df/hieradata/production.yaml#L55).